graphene_django/rest_framework/mutation.py

```diff
@@ -124,7 +124,7 @@ def mutate_and_get_payload(cls, root, info, **input):
         kwargs = cls.get_serializer_kwargs(root, info, **input)
         serializer = cls._meta.serializer_class(**kwargs)
 
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
             return cls.perform_mutate(serializer, info)
         else:
             errors = ErrorType.from_errors(serializer.errors)
```

Comment:
I would expect that this line would propagate the errors to the execution result (instead of needing to raise an exception). @no-dap do you know why it doesn't?

Reply:
I found this only happens with blank input in non-blankable field. I'm watching for what's going wrong now.

<img width="1328" alt="Screen Shot 2019-10-19 at 10 26 47 AM" src="https://user-images.githubusercontent.com/26595506/67135993-19c81700-f25b-11e9-8976-450f21ee1e9c.png">
(This works properly with an invalid input type)

Reply:
@jkimbo 
I found that when we append errors to MutationPayload class, works properly at first.  
<img width="1187" alt="Screen Shot 2019-10-19 at 11 15 37 AM" src="https://user-images.githubusercontent.com/26595506/67136507-0b312e00-f262-11e9-84ed-e261a3ff4894.png">
(The picture above is an ObjectType's `__init__` function)
  
But ObjectType initiated again to give some value for requested result, (which contains id)    
if we don't raise any exception.
<img width="1254" alt="Screen Shot 2019-10-19 at 11 19 04 AM" src="https://user-images.githubusercontent.com/26595506/67136535-7d097780-f262-11e9-9fd3-0c599ae33a5b.png">
I think this comes over the ErrorType, so we get an response instead of error messages.  
I'm not sure because I'm new on graphene,  
but I guess graphene's intend was raise exception  if somethings go wrong,  
rather than just provide error to execution result.

Reply:
I think we have to check about other mutations too.  
I'm on spare time now :)

Reply:
I think this is a graphene's bug.  
I could get an error message when I requested errors as query.  
<img width="1327" alt="Screen Shot 2019-10-19 at 12 31 41 PM" src="https://user-images.githubusercontent.com/26595506/67137454-96b1bb80-f270-11e9-8a67-0feb6a3db146.png">

Reply:
@jkimbo 
https://github.com/graphql-python/graphene-django/pull/803#issuecomment-544212540